google-cloud-compute-v1/acceptance/google/cloud/compute/v1/firewalls_smoke_test.rb

```diff
@@ -0,0 +1,42 @@
+require "minitest/autorun"
+
+require "google/cloud/compute/v1/firewalls"
+require "google/cloud/compute/v1/global_operations"
+
+# Tests for GCE firewalls
+class FirewallsSmokeTest < Minitest::Test
+  def setup
+    @default_project = ENV["COMPUTE_TEST_PROJECT"]
+    skip "COMPUTE_TEST_PROJECT must be set before running this test" if @default_project.nil?
+    @client = ::Google::Cloud::Compute::V1::Firewalls::Rest::Client.new
+    @client_ops = ::Google::Cloud::Compute::V1::GlobalOperations::Rest::Client.new
+    @name = "rbgapic#{rand 10_000_000}"
+    @firewalls = []
+  end
+
+  def teardown
+    @firewalls.each do |firewall|
+      @client.delete project: @default_project, firewall: firewall
+    end
+  end
+
+  def test_create_fetch
+    # we want to test here a field like IPProtocol
+    resource = {
+      name: @name,
+      source_ranges: ["0.0.0.0/0"],
+      allowed: [
+        {
+          I_p_protocol: "tcp",
+          ports: ["80"],
+        },
+      ],
+    }
+    operation = @client.insert project: @default_project, firewall_resource: resource
+    @client_ops.wait operation: operation.name, project: @default_project
```

Comment:
I think you need to call `.operation` on the original operation object to get the actual operation proto.  (See the other smoke tests.) The object that the client returns is a `Gapic::Rest::BaseOperation` wrapper. Are you sure this actually works as is?

Reply:
This PR was created the same day as https://github.com/googleapis/google-cloud-ruby/pull/13898 got merged that's why I missed "operation.operation".  It is fixed now.